### ISSUE TITLE
BUG - sev-3 - All - Messages: screenreader callout for unread messages exists/with variable

### ISSUE THREAD

**Issue by TKDickson** [department-of-veterans-affairs/va-mobile-app#8740]
## What happened?

We are (eventually) getting rid of the unread messages count on the category screen with home screen personalization.

In the meantime, a user with no unread messages is hearing "you have left brace left brace unread messages right brace right brace" from the screen reader on that button (or something like that, [here's the video](https://drive.google.com/file/d/1VExMjWFtZXeFQhmiDqosXE0WSj_BGF4h/view?usp=sharing)).

## Specs:

- Device:
- OS:
- User Account:
- Accessibility State:

## Desired behavior

As of the fix to #4982, we were announcing "you have no unread messages" or "you have 0 unread messages" in this scenario. Hopefully the H&B team will know for sure the desired behavior.

## Bug Severity - BE SURE TO ADD THE SEVERITY LABEL
 
See [Bug Tracking](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/QA#issue-severity) for details on severity levels

- Impact: High Low 
- Frequency: High Low 

- 3 - Low
- 2 - High
- 1 - Critical

## Screen shot(s) and additional information

<details>
  <summary>Full JSON response for services related to issue (expand/collapse)</summary>
  

</details>

## Ticket Checklist
- [ ] Steps to reproduce are defined
- [ ] Desired behavior is added
- [ ] Labels added (front-end, back-end, global, Health, relevant feature, accessibility, etc)
- [ ] Estimate of 1 added (for front-end tickets)
- [ ] Added either to the relevant feature epic (if found during new feature implementation), or the relevant team's bug epic ([Global](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8302), [Health & Benefits](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8303), [API](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8304), [QART](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8305))

**Comment by rbontrager** [comment]
@alexandec Before (I assume RQ) voiceOver said "you have zero unread messages" and not "you have no unread messages".  I feel like we should align with the old way.

**Comment by alexandec** [comment]
@rbontrager sure, I've changed to to "zero"

**Comment by rbontrager** [comment]
@alexandec I've caught one more issue (should hopefully be the last of them). If the user doesn't have SM authorization  (like user +119) then the screen reader used to just say "Messages" and not "You have zero unread messages" like it is with your changes. Including it in this ticket because a user that doesn't have SM authorization is also saying "you have left brace left brace unread messages right brace right brace" in dev currently.

**Comment by alexandec** [comment]
@rbontrager I think that authorization issue is separate but was covered up by the initial left brace/right brace issue. However I believe I have an approach where we can solve both, so we should be good. Going to increase the estimate slightly to account for that

**Comment by alexandec** [comment]
@rbontrager okay I've pushed my changes to the branch. That should fix the authorization issue. It should also fix the case where the call to the API endpoint fails for whatever reason

**Comment by rbontrager** [comment]
Verified that if the user does not have messages the screen readers says "you have zero unread messages". Verified that if the user does have unread message the screen reader says "you have [number] unread messages". Finally verified that if the user is not authorized then the screen reader says "Messages".  Approved by QA.
